app/models/foreman_tasks/triggering.rb

```diff
@@ -105,6 +105,7 @@ def parse_start_before!
 
     def can_start_recurring
       errors.add(:input_type, _('No task could be started')) unless recurring_logic.valid?
+      errors.add(:purpose, _('already exists for active or disabled state')) unless recurring_logic.valid_purpose?
```

Comment:
Currently I wouldn't be able to make sense out of it. Could we reword it a bit? Something along the lines of "Active or disabled recurring logic with purpose '$purpose' already exists".

![image](https://user-images.githubusercontent.com/7326770/130062029-dd2220c0-3421-4442-af1e-bb08d92a79c7.png)